### PR TITLE
Bt modules

### PR DESCRIPTION
**Pull request by jonaskuResearch**
Loop functions designed by Vincent van Pelt for his master thesis

### CODE REVIEW

File: loop-functions/CMakeLists.txt [hunk 1]
```diff
@@ -15,3 +15,6 @@ add_subdirectory(vanilla)
 #add_subdirectory(ModularityExperiment)
 #add_subdirectory(visual-add-ons)
 #add_subdirectory(localsearch)
+add_subdirectory(behaviortrees)
```
Comment (KenN7) [inline loop-functions/CMakeLists.txt]: This should not be activated by default, only vanilla should

Comment (KenN7) [reply loop-functions/CMakeLists.txt]: The rest is fine, once you've fixed that, lets merge

Comment (jonaskuResearch) [reply loop-functions/CMakeLists.txt]:
```suggestion
#add_subdirectory(behaviortrees)
```

**Comment by KenN7** [comment]
please delete the branch if not needed anymore

**Pull request merged**